src/utilities/pick.ts

```diff
@@ -0,0 +1,8 @@
+import {GeneralObject} from '../types';
+
+function pick(obj: GeneralObject | null, keyPaths: string[]) {
+  if (obj == null || keyPaths.length === 0) return {};
+  return keyPaths.reduce((acc, key) => ({...acc, [key]: obj[key]}), {});
```

Comment:
If a key path is passed that doesn’t exist in the object, it will return the key with value `undefined`. Is that your intent? Either way, I would add a test case for the expectation if a key path is not present in the object.